cfgov/cfgov/tests/test_urls.py

```diff
@@ -128,3 +128,14 @@ def test_handle_error(self, mock_render):
         mock_render.assert_called_with(
             request, '404.html', context={'request': request}, status=404
         )
+
+
+class TestBetaRefreshEndpoint(TestCase):
```

Comment:
I wasn't sure whether this is a good place to put these tests. Is there a better place to put them?